requirements.txt

```diff
@@ -1,4 +1,6 @@
 numpy==1.24.4
 sentencepiece==0.1.98
+torch>=2.0.0
```

Comment:
I don't think `torch` is needed

Reply:
It otherwise fails with a clean venv, so I needed to add it. I see the note in `convert.py` about dropping it as a dependency, but it is imported in a number of python files.

```
% find . -name '*.py' -not -path './tmp/*' -print0 | xargs -0 -- grep -nFe 'import torch'
./convert-lora-to-ggml.py:12:import torch
./convert-persimmon-to-gguf.py:1:import torch
./convert-hf-to-gguf.py:16:import torch
./examples/llava/llava-surgery.py:4:import torch
./examples/llava/convert-image-encoder-to-gguf.py:5:import torch
```

`./convert-lora-to-ggml.py` is the one that fails CI. I don't see any other python files being called in `run.sh`.

Reply:
Generally we have used `requirements.txt` for `convert.py` only. We also have a different `requirements-hf-to-gguf.txt` for `convert-hf-to-gguf.py`. I am not sure what changed, but `convert-lora-to-ggml.py` doesn't fail currently, but it does require pytorch.

Reply:
To clarify, `./convert-lora-to-ggml.py` is called in `run.sh`, and `run.sh` reliably fails with an error if torch is not included in the script's venv.

If it is not failing for you, you must already have pytorch installed in your user pip packages? Before my changes, `run.sh` did not use a virtual environment.

Instead of adding pytorch to `requirements.txt`, I can alternatively do a `pip install torch` in `run.sh` if that's preferable.